src/client/core/transport.js

```diff
@@ -1,81 +1,25 @@
 import * as hammerheadAPI from './deps/hammerhead';
```

Comment:
Agree, we should use `hammerhead`. But this refactoring will affect more than 30 files. Let's fix it in the https://github.com/superroma/testcafe-phoenix/issues/70 issue